TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithTruncateStatement.rdz.cbl

```diff
@@ -0,0 +1,14 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. DVZZMFT3.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+             EXEC SQL 
+              TRUNCATE TABLE INVENTORY
+              REUSE STORAGE
+              IGNORE DELETE TRIGGERS;
```

Comment:
Ok, the rdzSQL result is correct.
Please add more variants to test the syntax of DROP STORAGE, RESTRICT WHEN DELETE TRIGGERS and IMMEDIATE.